front-end/src/App.scss

```diff
@@ -22,6 +22,18 @@
   color: #61dafb;
 }
 
+.App-preloader {
```

Comment:
I suggest we use BEM in .block__element--modifier format everywhere. BEM is mentioned as highly recommended in the project requirements spec.

Reply:
These are temporary styles for the default component <App/>. For the Preloader component, all styles are written according to the methodology.

Reply:
all styles for the App component will be rewritten

Reply:
Got it